doc/api/inspector.md

```diff
@@ -224,6 +224,6 @@ session.post('HeapProfiler.takeHeapSnapshot', null, (err, r) => {
 
 [`'Debugger.paused'`]: https://chromedevtools.github.io/devtools-protocol/v8/Debugger#event-paused
 [`session.connect()`]: #inspector_session_connect
-[CPU Profiler]: https://chromedevtools.github.io/devtools-protocol/v8/Profiler
 [Chrome DevTools Protocol Viewer]: https://chromedevtools.github.io/devtools-protocol/v8/
-[Heap Profiler]: https://chromedevtools.github.io/devtools-protocol/v8/HeapProfiler
+[CPU Profiler]: https://chromedevtools.github.io/devtools-protocol/tot/Profiler
```

Comment:
On second look "tot" stays for tip of the tree while "v8" points to v8-inspector (node) so I think the original is more appropriate?

Reply:
I just checked again and looks like problem in uBlock Origin. At morning today I received text "404" on page and `tot` worked. Now I received 404 vs docs. Checked Firefox in incognito with adblocker and sometimes 404, sometimes all fine. Checked in Chrome in incognito (adblocker disabled) and old links works perfect all time.
I think I should close it, no changes required.
![Screenshot from 2019-09-26 22-11-30](https://user-images.githubusercontent.com/2633065/65717852-ef939700-e0aa-11e9-9f92-e84749047c62.png)

Thank you @lpinca